AAP-6551: Added info. about SECRET_KEY and -k setup option for installer (#1215)
RedHatInsights/red-hat-ansible-automation-platform-documentation: downstream/modules/platform/ref-setup-script-args.adoc

|`required_ram`|The minimum RAM required to install Tower (should only be changed for test installation)|`3750`
|`min_open_fds`|The minimum open file descriptions (should only be changed for test installations)|None
|`ignore_preflight_errors`|Ignore preflight checks, useful when installing into a template or other non-system image (overrides `required_ram` and `min_open_fds`)|`False`

> |`rekey_catalog: true`|When you execute the `setup.sh` script using this variable, by default, a new secret key is generated and distributed for both the automation controller and the automation services catalog.|`False`

Reviewer: ```suggestion
|`rekey_catalog`|When you execute the `setup.sh` script using this variable set to `true`, a new secret key is generated and distributed for both the automation controller and the automation services catalog.|`False`
```

Author: Thank you, Aine. I made the edit.
